stylesheets/blue/components/_nav.scss

```diff
@@ -249,5 +258,10 @@ $Nav-item-color-hover: $Theme-color-yellowOrange;
   .Nav-menu {
     display: flex;
     align-items: center;
+
+    padding-right: $Theme-spacing-small;
+    padding-left: $Theme-spacing-small;
+    margin-right: -$Theme-spacing-small;
+    margin-left: -$Theme-spacing-small;
```

Comment:
I can imagine a way where we wouldn't need the negative margin, which feels more natural to me. But feel free to merge it this way, it's ok with me, I just don't like negative margins :+1: